scene/resources/material.cpp

```diff
@@ -815,7 +815,17 @@ void BaseMaterial3D::_update_shader() {
 		code += "uniform float distance_fade_max;\n";
 	}
 
-	if (flags[FLAG_ALBEDO_TEXTURE_MSDF]) {
+	if (flags[FLAG_ALBEDO_TEXTURE_MSDF] && flags[FLAG_UV1_USE_TRIPLANAR]) {
+		if (textures[TEXTURE_ALBEDO].is_valid()) {
+			WARN_PRINT(vformat("%s (albedo %s): Texture msdf is not supported on triplanar materials. Ignoring texture msdf in favor of triplanar mapping.", get_path(), textures[TEXTURE_ALBEDO]->get_path()));
+		} else if (!get_path().is_empty()) {
+			WARN_PRINT(vformat("%s: Texture msdf is not supported on triplanar materials. Ignoring texture msdf in favor of triplanar mapping.", get_path()));
+		} else {
+			WARN_PRINT("Texture msdf is not supported on triplanar materials. Ignoring texture msdf in favor of triplanar mapping.");
```

Comment:
`Texture msdf` sounds odd, probably should be just `MSDF` or `MSDF rendering`.

```suggestion
			WARN_PRINT(vformat("%s (albedo %s): MSDF is not supported on triplanar materials. Ignoring MSDF in favor of triplanar mapping.", get_path(), textures[TEXTURE_ALBEDO]->get_path()));
		} else if (!get_path().is_empty()) {
			WARN_PRINT(vformat("%s: MSDF is not supported on triplanar materials. Ignoring MSDF in favor of triplanar mapping.", get_path()));
		} else {
			WARN_PRINT("MSDF is not supported on triplanar materials. Ignoring MSDF in favor of triplanar mapping.");
```

Reply:
Thanks!